pkgs/top-level/python-packages.nix

```diff
@@ -29468,4 +29468,97 @@ in modules // {
       sha256 = "14kb9gxw39zhvrijhp066b4bm6bgv35iw56c394y4dyczpha0dij";
     };
   };
+
+  pycalico = buildPythonPackage rec {
+    version = "0.16.0";
+    name = "pycalico-v${version}";
+    disabled = isPy3k;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "projectcalico";
+      repo = "libcalico";
+      rev = "v${version}";
+      sha256 = "0yxmdislw07wnv29xh81lp3d7gkq7176wa4ip91jpxsq5kiskga9";
+    };
+
+    doCheck = false;
+
+    propagatedBuildInputs = with self; [ netaddr python-etcd subprocess32 ];
+  };
+
+  calico = buildPythonPackage rec {
+    version = "1.4.0";
+    name = "calico-v${version}";
+    disabled = isPy3k;
+
+    src = pkgs.fetchFromGitHub {
+      owner = "projectcalico";
+      repo = "calico";
+      rev = "${version}";
+      sha256 = "1n7jijx05gys831acw41mwp5yfjlcjfvs80v58r7m18zvjddclys";
+    };
+
+    doCheck = false;
+
+    buildInputs = [ pkgs.makeWrapper ];
+
+    propagatedBuildInputs = with self; [
+      gevent greenlet netaddr python-etcd datrie ijson
+      msgpack pyparsing urllib3 posix-spawn prometheus_client
+    ];
+
+    patchPhase = ''
+      makeWrapper "${self.python}/bin/python" "$out/bin/felix-python" \
```

Comment:
so you want a Python interpreter that can find that package? For that, we have `python.buildEnv`/`python.withPackages`. Where do you use this `felix-python`?